### PR TITLE
🚣‍♀️ Add next.js example package

### PR DESCRIPTION
**Pull request by rzadp**

### CODE REVIEW

**Review by mj426382 — CHANGES_REQUESTED** [review]
Do we need vercel.svg and favicon.icon?

File: packages/example/tsconfig.json [hunk 1]
```diff
@@ -2,9 +2,11 @@
   "include": [
     "src",
     "types",
-    "playwright"
+    "playwright",
+    "src/assets/*"
```
[the remark this answers is not in the document]
Comment (rzadp) [reply packages/example/tsconfig.json]: Removed.

**Comment by rzadp** [comment]
> Do we need vercel.svg and favicon.icon?

Removed vercel.svg, replaced the favicon with our usedapp icon.

**Review by mj426382 — APPROVED** [review]
Everything looks good:)

**Pull request merged**